### PR TITLE
feat(monitoring): complete monitoring doc with new metrics

### PR DESCRIPTION
**Pull request by educhastenier**
* tomcat-specific metrics (community)
* cluster-specific metrics (subscription cluster)

### CODE REVIEW

File: md/runtime-monitoring.md [hunk 1]
```diff
@@ -152,3 +173,31 @@ Sample extract of exposed Prometheus data:
...
+
+### Cluster-specific metrics
+
+::: info
+**Note:** For Enterprise, Performance editions only, with cluster mode activated.
+:::
+
+A Bonita 7.10+ cluster has monitoring on distributed cluster locks ACTIVE by default.
+
+To DISABLE those metrics, simply edit file
```
Comment (danila-m) [inline md/runtime-monitoring.md]:
```suggestion
To DISABLE those metrics, simply edit the file
```

File: md/runtime-monitoring.md [hunk 2]
```diff
@@ -66,13 +67,18 @@ be changed (property `org.bonitasoft.engine.monitoring.logging.step`).
 
 ::: info
 To change any value, **uncomment the line by removing the # character**, and change the true / false value.  
-Server restart is required for the changes to take effect.
+Then push your configuration changes to database:
+```bash
+./setup/setup.sh push
+```
+Then restart Tomcat server for the changes to take effect.
```
Comment (danila-m) [inline md/runtime-monitoring.md]:
```suggestion
Then restart the Tomcat server for the changes to take effect.
```

File: md/runtime-monitoring.md [hunk 3]
```diff
@@ -93,15 +101,23 @@ They provide information about:
...
 Each of these metrics provides many different counters to finely understand what is going on.
 
 ::: info
 To change any value, **uncomment the line by removing the # character**, and change the true / false value.  
-Server restart is required for the changes to take effect.
+Then push your configuration change to database:
+```bash
+./setup/setup.sh push
+```
+Then restart Tomcat server for the changes to take effect.
```
Comment (danila-m) [inline md/runtime-monitoring.md]:
```suggestion
Then restart the Tomcat server for the changes to take effect.
```

File: md/runtime-monitoring.md [hunk 4]
```diff
@@ -122,7 +138,12 @@ to
     # Activate publication of metrics to prometheus:
     com.bonitasoft.engine.plugin.monitoring.prometheus.enable=true
 
-and restart server.
+Then push your configuration change to database:
+```bash
+./setup/setup.sh push
+```
+Then restart Tomcat server for the changes to take effect.
```
Comment (danila-m) [inline md/runtime-monitoring.md]:
```suggestion
Then restart the Tomcat server for the changes to take effect.
```

**Review by danila-m — APPROVED** [review]
Good Job. :+1: 
Typos/english

**Comment by tnguyen1** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

**Pull request merged**